### PR TITLE
Fix Flake8 violations

### PR DESCRIPTION
**Pull request by dorothykiz1**
To fix Flake8 violations for ;
1. asv/__init__.py
2. asv/config.py

### CODE REVIEW

File: asv/__init__.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+
```
Comment (datapythonista) [inline asv/__init__.py:7]: I don't think this is required by flake8, is it?

Comment (dorothykiz1) [reply asv/__init__.py:7]: No, it's not, this has been fixed. Thank You, Marc

File: asv/__init__.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-
+
```
Comment (datapythonista) [inline asv/__init__.py:6]: I think flake8 should be complaining about this. Are you running flake8 at all to check your changes?

**Comment by datapythonista** [comment]
CI is red. You also need to merge master into your branch, and run flake8 in the CI. You can use #988 as an example.

Same for the rest of PRs.

**Comment by dorothykiz1** [comment]
To be revised first

**Pull request closed without merging**